converters/fromZigbee.js

```diff
@@ -40,6 +40,26 @@ const toPercentage = (value, min, max) => {
     return (normalised * 100).toFixed(2);
 };
 
+const toPercentageCR2032 = (voltage) => {
+    let Percentage = null;
+    
+    if (voltage >= 3000) {
+		Percentage = 100;
+    } else if (voltage < 3000) {
+        Percentage = 100 - ((3000 - voltage) * 58) / 100;
+    } else if (voltage < 2900) {
```

Comment:
I'm unfamiliar with the inner workings of javascript but wouldn't only the first two clauses be used? `voltage < 2900` is **also** `< 3000`. In C the order must be changed (going up from `< 2100`).

Reply:
You where right. I changed the order.